packages/cli/src/utils/getNpmPackages.ts

```diff
@@ -0,0 +1,34 @@
+import fetch from "node-fetch";
+
+export const getNpmPackages = async (): Promise<
+  { name: string; value: string }[]
+> => {
+  try {
+    const response = await fetch(
```

Comment:
This library is a polyfill.
Since Node 18, fetch has been supported at the Node engine level.
Is there a specific reason why we need to use a polyfill in this case?

If it's for backward compatibility, may I ask why this specific polyfill was chosen?

Reply:
fix it to axios. thx XD